include/rpc/des.h

```diff
@@ -61,7 +61,7 @@ struct desparams {
 /*
  * These ioctls are only implemented in SunOS. Maybe someday
  * if somebody writes a driver for DES hardware that works
- * with FreeBSD, we can being that back.
+ * with FreeBSD, we can bring that back.
```

Comment:
DES is deprecated. Why should we fix a typo here as opposed to maybe deleting it entirely? FreeBSD at this point is not at all likely to add these IOCTLs, I don't think.

Reply:
This needs to be rebased now. emaste removed all this code a few hours ago.